jenkins/bootstrap.py

```diff
@@ -770,6 +770,7 @@ def setup_root(call, root, repos, ssh, git_cache, clean):
         os.makedirs(root)
     root_dir = os.path.realpath(root)
     logging.info('Root: %s', root_dir)
+    os.environ['GIT_TRACE'] = '1'
```

Comment:
I mean isn't it will pollute os.environ when I run bootstrap locally?

I think you are only interested in the `git fetch` call, make it `GIT_TRACE=1 git fetch`?

And I think get the ssh key from Jenkins and test locally might be easier, just my thought.